bot/cogs/games/tic_tac_toe.py

```diff
@@ -0,0 +1,296 @@
+import random
```

Comment:
Global comment on this file : use `discord.ext.menus` instead of your listener. Your code will be much more readable

Reply:
Alright, I'll look into it, note that you can post a global comment  while reviewing like this:
![Peek 2020-07-27 13-41](https://user-images.githubusercontent.com/20902250/88538073-037f9080-d00f-11ea-9d3b-21b3f6be6815.gif)

Reply:
yes